bundle.yaml

```diff
@@ -1,21 +1,85 @@
+# The machine constraints for each service in this bundle
+# have been commented out so you don't run into quota
+# problems on public clouds. Modify and uncomment the
+# constraints: lines for each service to reflect your
+# deployment before moving to production.
+#
 series: xenial
-services:
-  kubernetes:
-    charm: cs:xenial/kubernetes-8
-    options:
-      version: v1.3.4
+machines:
+  0:
+  1:
+  2:
+  3:
+applications:
+  "kubernetes-master":
+    charm: "cs:~containers/kubernetes-master-3"
```

Comment:
We **just** incremented the version of the kubernetes-master charm last night. s/3/4/